app/src/main/java/com/serranoie/android/buybuddy/ui/common/CategoryCard.kt

```diff
@@ -33,9 +33,11 @@ import androidx.compose.ui.unit.dp
 import androidx.navigation.NavController
 import com.serranoie.android.buybuddy.data.persistance.entity.CategoryWithItemsEntity
 import com.serranoie.android.buybuddy.ui.navigation.Route
+import com.serranoie.android.buybuddy.ui.util.UiConstants.CONTENT_ANIMATION_DURATION
 import com.serranoie.android.buybuddy.ui.util.UiConstants.basePadding
 import com.serranoie.android.buybuddy.ui.util.UiConstants.largePadding
 import com.serranoie.android.buybuddy.ui.util.UiConstants.smallPadding
+import com.serranoie.android.buybuddy.ui.util.noRippleClickable
```

Comment:
No need to implement the `view.weakHapticFeedback()` since it's an UI change inside the Material Design 3 system, trying to delete the ripple effect when the user taps into the item itself.

So in this case it was a coincidence that `SummaryScreen.kt` and `CategoryCard.kt` had the same `view.weakHapticFeedback()` callback when the `noRippleClickable` was also called.

Marking this as resolved.